Mathlib/Topology/PartitionOfUnity.lean

```diff
@@ -302,7 +302,8 @@ theorem exists_finset_nhd_support_subset {U : ι → Set X} (hso : f.IsSubordina
 `fun x ↦ ∑ᶠ i, f i x • g i x` is a continuous function. -/
 theorem IsSubordinate.continuous_finsum_smul [ContinuousAdd E] {U : ι → Set X}
     (ho : ∀ i, IsOpen (U i)) (hf : f.IsSubordinate U) {g : ι → X → E}
-    (hg : ∀ i, ContinuousOn (g i) (U i)) : Continuous fun x => ∑ᶠ i, f i x • g i x :=
+    (hg : ∀ i, ContinuousOn (g i) (U i)) :
+    Continuous fun x => ∑ᶠ i, f i x • g i x :=
```

Comment:
Why this change?

Reply:
But it doesn't seem like

Reply:
But it doesn't seem like you are changing anything on this line, so it should not go past the 100 chars?